Content.Shared/Ninja/Systems/NinjaSuitSystem.cs

```diff
@@ -0,0 +1,154 @@
+using Content.Shared.Inventory.Events;
+using Content.Shared.Ninja.Components;
+using Content.Shared.Stealth;
+using Content.Shared.Stealth.Components;
+using Content.Shared.Timing;
+using Robust.Shared.GameStates;
+using Robust.Shared.Network;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Ninja.Systems;
+
+public abstract class SharedNinjaSuitSystem : EntitySystem
+{
+    [Dependency] private readonly SharedNinjaGlovesSystem _gloves = default!;
+    [Dependency] protected readonly SharedNinjaSystem _ninja = default!;
+    [Dependency] private readonly SharedStealthSystem _stealth = default!;
+    [Dependency] protected readonly UseDelaySystem _useDelay = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<NinjaSuitComponent, GotEquippedEvent>(OnEquipped);
+        SubscribeLocalEvent<NinjaSuitComponent, ComponentGetState>(OnGetState);
+        SubscribeLocalEvent<NinjaSuitComponent, ComponentHandleState>(OnHandleState);
+        SubscribeLocalEvent<NinjaSuitComponent, GotUnequippedEvent>(OnUnequipped);
+
```

Comment:
```suggestion
        SubscribeLocalEvent<NinjaSuitComponent, GotUnequippedEvent>(OnUnequipped);
        SubscribeLocalEvent<NinjaSuitComponent, ItemStatusCollectMessage>(OnNinjaSuitItemStatus);

```

I suggest adding a readout to the ninja suit via item status. I know you have a power alert, but having a supplemental number that the ninja could read would be nice. For example, syringes show how much solution is remaining when it's in your active hand. 

On the other hand, cells show how much % of a charge they have when you examine them. So either option could be a way to go here. If you want to look more into ItemStatus, either look at Syringes or Implanters. Or if you want to see how cells do it on examine, check out cells.

If you do go the item status route, you'll need this event, and a UI component. When implemented this should appear above the suit. Otherwise I say add a % readout to the alert when examined.

Reply:
im fairly sure there is a thing for relaying examine from container to contained power cell, ill see if i can use that first since syringes show the ui **when in hand** not when equipped, would be an unnecessary hurdle

Reply:
have done it on examine

Reply:
Update on this: My mistake, you can't examine the UI element. You should be able to add it to the element when you hover over it.

Reply:
not sure how to add % to the alert, but doing it on the suit itself was easy enough